### PR TITLE
Feature/PC console - Add SAC to query search

### PR DESCRIPTION
**Pull request by xkopenreview**
This pr should fix the following 2 issues:
1. search value is converted to lowercase so "==" in query search may not work
2. to add sac to query search in pc console

### CODE REVIEW

**Review by zbialecki — COMMENTED** [review]
Is it required that we have separate filters for case sensitive and case insensitive search? I can't think of a case where case sensitive search is required.

File: unitTests/webfield-utils.test.js [hunk 1]
```diff
@@ -523,6 +523,70 @@ describe('filterCollections', () => {
...
+      collections,
+      filterString,
+      filterOperators,
+      propertiesAllowed,
+      uniqueIdentifier
+    )
+    expect(result.filteredRows.map((p) => p.id)).toEqual([2, 3])
+
+    // exact match
+    filterString = 'sac=="Name two"'
```
Comment (melisabok) [inline unitTests/webfield-utils.test.js:569]: Can we filter by tilde id too?

**Comment by xkopenreview** [comment]
> Is it required that we have separate filters for case sensitive and case insensitive search? I can't think of a case where case sensitive search is required.

= is includes search
while == is for exact search so i think the filtering should not do any case conversion, it should compare with the exact string given by the user

File: unitTests/webfield-utils.test.js (hunk 1 above)
Comment (xkopenreview) [reply unitTests/webfield-utils.test.js:584]: yes, added in [98e23a1](https://github.com/openreview/openreview-web/pull/1830/commits/98e23a19285461925879021875bd46a4828576fc)

**Comment by zbialecki** [comment]
> = is includes search

OK, that makes sense.

**Review by zbialecki — APPROVED** [review]

**Review by melisabok — APPROVED** [review]

**Pull request merged**